src/components/builder/derive-address.tsx

```diff
@@ -0,0 +1,25 @@
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {Checkbox} from '../formio';
+
+const DeriveAddress = () => {
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'DeriveAddress' builder field",
+    defaultMessage: 'Derive address.',
```

Comment:
You can be a bit more inspiring here :wink: 

```suggestion
    defaultMessage: 'When enabled, the street name and city are derived from the entered postcode and house number.',
```

be informative, provide useful context, don't just repeat the label, otherwise we can just leave out the tooltip entirely.